README.md

```diff
@@ -1 +1,37 @@
-# training-hello-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Hello Microverse
+
```

Comment:
I think it will be better if here you put a short description of the project instead of `Hello Microverse`

Reply:
@cynthiainga project description updated